snuba/migrations/system/0001_migrations.py

```diff
@@ -0,0 +1,29 @@
+from typing import Sequence
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migrations
+from snuba.migrations import operations
+
+
+class Migration(migrations.Migration):
+    is_dangerous = False
+    dependencies = None
+
+    def forwards_local(self) -> Sequence[operations.Operation]:
+        return [
+            operations.RunSql(
+                storage_set=StorageSetKey.MIGRATIONS,
+                statement="""\
+                    CREATE TABLE IF NOT EXISTS migrations_local (app String,
+                    migration_id String, timestamp DateTime, status Enum('completed' = 0,
+                    'in_progress' = 1), consumer_offsets String)
```

Comment:
If we do not have a structure for the consumer_offset I would not add the column just yet. It will be easier to introduce the column later than changing the type from string to something more structured.